app/brave_generated_resources.grd

```diff
@@ -146,6 +146,22 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_APPEARANCE_SETTINGS_BRAVE_THEMES" desc="The label for brave theme change setting options">
         Brave colors
       </message>
+      <!-- Tor -->
+      <message name="IDS_NEW_TOR_IDENTITY" desc="The text label of a menu item for requesting new tor identity">
+        New Tor Identity
+      </message>
+      <message name="IDS_PROFILES_EXIT_TOR" desc="Button in the avatar menu bubble view for exiting the active tor session.">
+        Exit Tor
+      </message>
+      <message name="IDS_TOR_PROFILE_NAME" desc="The name given to the tor browsing profile. Displayed in the avatar menu bubble and button UI.">
+        Tor
+      </message>
+      <message name="IDS_PROFILES_OPEN_TOR_PROFILE_BUTTON" desc="Button in the avatar menu bubble view to open a tor window.">
+        Open Tor Window
+      </message>
+      <message name="IDS_PROFILES_TOR_PROFILE_NAME" desc="Name of the tor profile.">
```

Comment:
nit: I know it's only a description, but capitalization might help a localizer to know it's a name.